pkg/controller/main.go

```diff
@@ -466,6 +466,19 @@ func NewIPVSController(kubeClient *kubernetes.Clientset, namespace string, useUn
 			return
 		}
 
+		state := ipvsc.keepalived.isMaster()
```

Comment:
I don't think it's appropriate that invoking the health check endpoint can somehow trigger pod labels to be changed. This logic should live elsewhere.